presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -216,6 +217,20 @@ public static double atan2(@SqlType(StandardTypes.DOUBLE) double num1, @SqlType(
         return Math.atan2(num1, num2);
     }
 
+    @Description("Binomial cdf given n, prob, and a value")
```

Comment:
what is n? I assume `prob` is `probability`? Can we make it more novice reader friendly (also the variable names).